app/decorators/assignation_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class AssignationDecorator < Draper::Decorator
+  delegate_all
+
+  def display_name
+    "#{object.schedule.user.username}: #{object.term.display_name}"
+  end
+end
```

Comment:
Wydaje mi się, że jest - za ilość plików się nie płaci, a zyskuje się to, że metody powiązane z prezentacją są w osobnym miejscu od modelu.